FHEM/14_SD_WS.pm

```diff
@@ -443,8 +443,40 @@ sub SD_WS_Parse($$)
 				temp       => sub {my (undef,$bitData) = @_; return ((SD_WS_binaryToNumber($bitData,12,23) - 500) / 10.0); },
 				hum        => sub {my (undef,$bitData) = @_; return SD_WS_binaryToNumber($bitData,24,31); },
 				crcok      => sub {return 1;},		# crc test method is so far unknown
-			} ,
-    );
+			} ,	
+		94 => {			
+				# Sensor sends Bit 0 as "0", Bit 1 as "110"
+				# Protocol after conversion bits (Length varies from minimum 24 to maximum 32 bits.)
+				# ------------------------------------------------------------------------
+				# 0    4    | 8    12   | 16   20   | 24   28
+				# 1111 1100 | 0000 0110 | 0001 0000 | 0011 0111
+				# iiii iiii | ??s? tttt | tttt tttt | ???? ????
+				# i:  8 bit id
+				# s:  1 bit sign (0 = temperature positive, 1 = temperature negative
+				# t: 12 bit temperature (MSB-first, BCD, 12 bit unsigned celsius scaled by 10)
+				# ?: unknown
+				sensortype => 'Atech',
+				model      => 'SD_WS_94_T',
+				prematch   => sub { return 1; },		#  no precheck known
+				id         => sub { # change 110 to 1 in ref bitdata and return id
+									($_[1] = $_[1]) =~ s/110/1/g; 
+									return sprintf('%02X', SD_WS_bin2dec(substr($_[1],0,8))); 
+								  },  
+				bat        => sub { return substr($_[1],8,1) eq "0" ? "ok" : "low"; },
```

Comment:
Wie kommst du darauf, das es ein Batteriebit gäbe? Hast du eine Doku zum Protokoll gefunden? Im Issue [Atech wireless weather station](https://github.com/RFD-FHEM/RFFHEM/issues/547) findet sich auch kein Hinweis dazu.

Reply:
Hab da wohl zu viel kopiert. Ich werde es wieder entfernen